docs/reference/action_list.rst

```diff
@@ -801,3 +841,76 @@ Combine this with configuring a custom template and you'll have a list column fu
 
 .. _`SonataDoctrineORMAdminBundle Documentation`: https://docs.sonata-project.org/projects/SonataDoctrineORMAdminBundle/en/4.x/reference/list_field_definition/
 .. _`here`: https://github.com/sonata-project/form-extensions/tree/1.x/src/Type
+
+Advance Usage
+-------------
+
+Displaying sub entity properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to display only one field from a sub entity or embedded object in a dedicated column, you can simply use the dot-separated notation::
+
+    namespace App\Admin;
+
+    use Sonata\AdminBundle\Admin\AbstractAdmin;
+    use Sonata\AdminBundle\Form\FormMapper;
+    use Sonata\AdminBundle\Datagrid\DatagridMapper;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Sonata\AdminBundle\Show\ShowMapper;
+
+    final class UserAdmin extends AbstractAdmin
+    {
+        protected function configureListFields(ListMapper $list): void
+        {
+            $list
+                ->addIdentifier('id')
+                ->addIdentifier('firstName')
+                ->addIdentifier('lastName')
+                ->addIdentifier('address.street')
+                ->addIdentifier('address.ZIPCode')
+                ->addIdentifier('address.town')
+            ;
+        }
+    }
+
+.. note::
+
+    This only makes sense when the prefix path is made of entities, not collections.
+
+Custom template
+^^^^^^^^^^^^^^^
+
+If you need a specific layout for a row cell, you can define a custom template::
+
+    namespace Sonata\MediaBundle\Admin;
+
+    use Sonata\AdminBundle\Admin\AbstractAdmin;
+    use Sonata\AdminBundle\Form\FormMapper;
+    use Sonata\AdminBundle\Datagrid\DatagridMapper;
+    use Sonata\AdminBundle\Datagrid\ListMapper;
+    use Sonata\AdminBundle\Show\ShowMapper;
+
+   final class MediaAdmin extends AbstractAdmin
+    {
+        protected function configureListFields(ListMapper $list): void
+        {
+            $list
+                ->addIdentifier('id')
+                ->add('image', 'string', ['template' => '@SonataMedia/MediaAdmin/list_image.html.twig'])
+                ->add('custom', 'string', ['template' => '@SonataMedia/MediaAdmin/list_custom.html.twig'])
```

Comment:
Should we use here `FieldDescriptionInterface::TYPE_STRING` constant? instead of `string`

Reply:
Yes,  we are using this above!

Thank you @franmomu